### PR TITLE
Added new "cul8r " slang to the data base

### LINKED ISSUE #244: cul8r is not present in the DB
### Description

# cul8r: see you later
This shortcut shorthand is not added to the database

### PR DESCRIPTION
**Pull request by JasonDsouza212**
Closes #244

### What new changes did you make? Tick all applicable boxes
- [ ✓] Added new abbreviation
- [ ] Fixed something in the source code
- [ ] Added a new feature
- [ ] Fixed the docs (README.md, CONTRIBTUING.md, etc)

### Describe the new changes you added.
Added a new slang **cul8r** which means See You Later

### CODE REVIEW

File: public/server/db.json [hunk 1]
```diff
@@ -470,5 +470,8 @@
   },
   "dns": {
     "definition": "Domain Name Service"
+  },
```
Comment (Njong392) [inline public/server/db.json:473]: You didn't need to add this to the `db.json` anymore.

File: public/server/db/cul8r.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "definition": "See You Later"
```
Comment (Njong392) [inline public/server/db/cul8r.json:2]: This here looks amazing!

**Review by Njong392 — CHANGES_REQUESTED** [review]
Thank you for the changes, but I see that you have made changes to the `package-lock.json` file. To revert this, please check out [this guide](https://gist.github.com/kumarsonsoff3/fbc6ecde519250a9f0a4d7ccc9993a1a).

To avoid this, instead of running `npm install` to install dependencies for this project, use `npm ci` as stated and referenced in the README.

**Comment by JasonDsouza212** [comment]
I reverted the **package-lock.json** and even removed cul8r from **db.json** . I hope everything seems fine now

**Review by Njong392 — APPROVED** [review]
Perfect! Thank you so much for the contribution 💟

**Pull request merged**